app/main.py

```diff
@@ -14,4 +13,9 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+
+    counter = 0
+    for char in phrase:
+        if char.lower() == letter.lower():
+            counter += 1
+    return counter
```

Comment:
yeah, it's a working solution, but try to use .count() method of string, it'll look better ;)